src/wallet/wallet.h

```diff
@@ -942,6 +942,7 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
 
     //! Returns all unique ScriptPubKeyMans in m_internal_spk_managers and m_external_spk_managers
     std::set<ScriptPubKeyMan*> GetActiveScriptPubKeyMans() const;
+    bool IsActiveScriptPubKeyMan(const ScriptPubKeyMan& spkm) const;
```

Comment:
tiny nit:
To follow the same pattern used across this class, should pass the spkm pointer instead of the const ref (and return early if it is nullptr).

Side note:
It is not really something that we should tackle nor discuss here but I think it would be better to return the is_active, and other spkm information, on the initial lookup function instead of introducing multiple functions that retrieves the information after wise. E.g. `GetAllScriptPubKeyMansInfo()` could return a struct instead of the bare spkm pointers.

Reply:
It's probably not a great idea to be passing around raw pointers and we should be moving away from doing that.